how-to/hardware-wallets/ledger/how-to-recover-assets-sent-to-the-wrong-address.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to recover assets sent to the wrong address
+tags:
+  - Ledger
+date_published: '2021-01-14'
+date_modified: '2021-01-14'
+---
+
+If you, for example, sent ETH to your ETC address, or EWT to your ETH address, you cannot simply transfer it back when using a hardware device like a Ledger wallet, due to restrictions in the standard apps provided by Ledger.
+
+One way to recover those funds is to use the recovery phrase for your Ledger device, but this comes with the risk that your recovery phrase may be stolen by malware or phishing. To access those coins in a safer way, that does not involve using the recovery phrase, you can use the ETH Recovery app provided by Ledger. Simply follow the steps below.
+
+## Install the ETH Recovery app
+
+<Alert>
+
+The ETH Recovery app is meant for recovering assets only. You **should not** use this app on a day to day basis, for transferring funds.
```

Comment:
Note: does this article and recovery app only apply to EVM-based chains? If so, please note that.